Sources/AccountSDKIOSWeb/Lib/Storage/Keychain/Compat/LegacyKeychainSessionStorage.swift

```diff
@@ -65,4 +66,13 @@ class LegacyKeychainSessionStorage {
 
         return try? JSONSerialization.jsonObject(with: jws.payload.data()) as? [String: Any]
     }
+    
+    private func validateTokenFormat(_ token: String) -> String {
+        var validToken = token
+        if !validToken.starts(with: "e") {
```

Comment:
Does all valid tokens always start with an e?

Reply:
Yes, because the token string is basically a base64 encoded string. And a "{" in base64 starts with "e". Please check it out here: [https://www.base64decode.org/](https://www.base64decode.org/) with an example header from our JWS token _eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9_ To correctly decode it by JOSE library it must be valid.

Reply:
This is so weird that the old SDK changes the first character to a random letter...